app/templates/components/orange-bar-custom-map-study.hbs

```diff
@@ -74,7 +74,9 @@
         </li>
       </ul>
     </div>
-    {{!-- <li class="menu align-right"><a href="#">Right Side Placeholder</a></li> --}}
+    {{!-- <li class="menu align-right"><a href="#">Right Side Placeholder</a></li>
```

Comment:
Prefer deleting this here, unless it's meant to hold a shapefile feature or something?